packages/SwingSet/misc-tools/extract-transcript-from-kerneldb.js

```diff
@@ -30,29 +40,18 @@ if (!vatName) {
   console.log(`all vats:`);
   for (const name of allVatNames) {
     const vatID = get(`vat.name.${name}`);
-    const startPos = Number(get(`${vatID}.t.startPosition`));
-    const endPos = Number(get(`${vatID}.t.endPosition`));
-    const len = `${endPos - startPos}`;
     const status = `(static)`;
-    console.log(
-      `${vatID.padEnd(3)} : ${name.padEnd(15)} ${status.padEnd(
-        20,
-      )} ${len.padStart(10)} deliveries`,
-    );
+    console.log(`${vatID.padEnd(3)} : ${name.padEnd(15)} ${status.padEnd(20)}`);
   }
   for (const vatID of allDynamicVatIDs) {
-    const startPos = Number(get(`${vatID}.t.startPosition`));
-    const endPos = Number(get(`${vatID}.t.endPosition`));
-    const len = `${endPos - startPos}`;
     const options = JSON.parse(get(`${vatID}.options`));
     const { name, managerType } = options;
     const status = `(dynamic, ${managerType})`;
-    console.log(
-      `${vatID.padEnd(3)} : ${name.padEnd(15)} ${status.padEnd(
-        20,
-      )} ${len.padStart(10)} deliveries`,
-    );
+    console.log(`${vatID.padEnd(3)} : ${name.padEnd(15)} ${status.padEnd(20)}`);
   }
+} else if (/(supervisor|lockdown)(B|-b)undle/.test(vatName)) {
```

Comment:
is this abusing the CLI argument so that when you say "please replay the vat named supervisor-bundle", it will actually extract the given bundle and write it to disk?

weird, but I'm ok with it

Reply:
I thought I had removed this, since it's been included in the extract-bundle tool instead...